static/js/base.js

```diff
@@ -57,6 +75,10 @@ function stop() {
     /*
     Stop the robot thread
     */
-    simulator.stop();
-    update({x:144, y:144, dir:0});
+    worker.terminate();
+    worker = new Worker("/static/js/robot.js");
+    worker.onmessage = onmessage;
+    worker.postMessage({code:code});
```

Comment:
As discussed, we should try not to have to spawn a new worker each time, since every new worker needs to load Pyodide. I think it may actually not be possible, but we can look into it more. If it isn't possible, we could try to make it so that we only create a new worker if we're in the middle of executing autonomous (so not during teleop/idle).